src/containers/Login/style.module.css

```
@@ -0,0 +1,46 @@
+.container {
+  width: 100vw;
+  height: 100vh;
+
+  background-color: white;
+
```

remover espaços em branco dentro da classe 
```suggestion
  background-color: white;
```